hack/tools/Makefile

```diff
@@ -1,4 +1,4 @@
-# Copyright 2021 The Kubernetes Authors.
+# Copyright 2020 The Kubernetes Authors.
 #
 # Licensed under the Apache License, Version 2.0 (the "License");
 # you may not use this file except in compliance with the License.
```

Comment:
I had to do the following to get the `make serve` target in the book folder to work:
```bash
cd ./hack/tools
go get -v  -tags=tools sigs.k8s.io/cluster-api/hack/tools/mdbook/embed@v0.0.0-20210313163703-752c727cf58b
```

This lead to a new entry in hack/tools/go.sum
```bash
sigs.k8s.io/kubebuilder/docs/book/utils v0.0.0-20200226075303-ed8438ec10a4 h1:P/Vxe8zHHI0mjkl9+5UuWJgynJgLxoVpisfWKWr3zl4=
```

Reply:
`go mod tidy` will remove this line ...

Reply:
go get -v  -tags=tools sigs.k8s.io/cluster-api/hack/tools/mdbook/embed@v0.0.0-20210313163703-752c727cf58b

will be removed by `go mod tidy`

I guess I need update go.mod first but this doesn't exist on aws provider ..

Reply:
Try adding the following line to hack/tools/tools.go
```go
_ "sigs.k8s.io/cluster-api/hack/tools/mdbook/embed"
```

CAPA Is probably still using Go < 1.16, and that's why they don't get the error